tests/test_worker.ts

```diff
@@ -710,6 +709,41 @@ describe('workers', function () {
     await worker.close();
   });
 
+
+  describe('when prioritized job is added while processing last active job', () => {
+    it('should process prioritized job whithout delay', async function () {
```

Comment:
without (typo)

Reply:
Its a typo from the original code but sure I can fix it in our branch